hadoop-hdds/docs/content/feature/Decommission.md

```diff
@@ -51,6 +51,14 @@ ozone admin datanode decommission [-hV] [-id=<scmServiceId>]
 ```
 You can enter multiple hosts to decommission multiple datanodes together.
 
+To view the status of a decommissioning datanode, you can execute the following command in cli,
+
+```shell
+ozone admin datanode status decommission [-hV] [-id=<scmServiceId>] [--scm=<scm>] [--id=<uuid>] [--ip=<ipAddress>]
+```
+You can pass the ipAddress or uuid of one datanode to view only the details related to that datanode.
```

Comment:
```suggestion
You can pass the IP address or UUID of one datanode to view only the details related to that datanode.
```

Reply:
I have incorporated these suggestions in the recent commit. Thank you for the review!